extensions/tls-registry/spi/src/main/java/io/quarkus/tls/TrustAllOptions.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkus.tls.runtime.keystores;
+package io.quarkus.tls;
```

Comment:
This is considered API?

Reply:
This one is a bit annoying. It's not API per se, but it's convenient to have it here, as someone implementing its own keystore may want to use a trust-all behavior (without having to implement it).